maint/apply-code-format.sh

```diff
@@ -36,24 +36,8 @@ else
     SED_FLAGS='-rn'
 fi
 
-# Find out python version. Use yapf only when in Python 3
-if PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[:][0])')
-then
-    if [[ "3" != "${PYTHON_VERSION}" ]]; then
-        echo "Python reports version number '${PYTHON_VERSION}' so will skip yapf formatting. Please use Python3"
-    fi
-else
-    bash_lib_print_error "Failed invocation of Python."
-    exit 1
-fi
-
-
 declare CLANG_FORMAT_BASENAME="clang-format-3.9"
 declare REQUIRED_CLANG_FORMAT_VERSION=3.9
-if [[ "3" == "${PYTHON_VERSION}" ]]; then
-    declare YAPF_FORMAT_BASENAME="yapf"
-    declare REQUIRED_YAPF_FORMAT_VERSION=0.26
```

Comment:
right now its not asserted if its yapf 0.26 or some other version. Which means that everytime yapf releases a new version we might have to commit a new change that reformats python code according to the latest version of yapf. Which is not a bad idea: then the idea is that our repo is always formatted by latest yapf.

in that case, we need to make changes in lines like this:
https://github.com/NervanaSystems/ngraph-tf/blob/dc93c49be6a98e0f149c914c25f0280e88f3488f/test/ci/docker/docker-scripts/run-check-code-format.sh#L129
https://github.com/NervanaSystems/ngraph-tf/blob/dc93c49be6a98e0f149c914c25f0280e88f3488f/test/ci/docker/dockerfiles/Dockerfile.ngraph-tf-ci-py2#L63
https://github.com/NervanaSystems/ngraph-tf/blob/dc93c49be6a98e0f149c914c25f0280e88f3488f/test/ci/docker/dockerfiles/Dockerfile.ngraph-tf-ci-py3#L67

Reply:
Agree. We will need to add that in a separate PR.